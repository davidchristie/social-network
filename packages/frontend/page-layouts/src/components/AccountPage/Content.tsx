import { Container, Section } from "design-system";
import { UpdateAccountForm, UpdatePasswordForm, UpdateProfileForm } from "domain-model";
import React from "react";

const Content: React.StatelessComponent = () => {
  return (
    <Container>
      <Section>
        <h1>Account</h1>
        <UpdateProfileForm />
        <UpdateAccountForm />
        <UpdatePasswordForm />
      </Section>
    </Container>
  );
};

export default Content;
