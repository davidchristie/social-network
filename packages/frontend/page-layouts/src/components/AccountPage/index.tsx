import React from "react";
import Page from "../Page";

const AccountPage = () => {
  return <Page load={() => import("./Content")} />;
};

export default AccountPage;
