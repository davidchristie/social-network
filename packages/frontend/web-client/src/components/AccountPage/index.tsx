import { Container, Section } from "design-system";
import { UpdateAccountForm, UpdatePasswordForm, UpdateProfileForm } from "domain-model";
import React from "react";

export default class AccountPage extends React.Component {
  public render () {
    return (
      <Container>
        <Section>
          <h1>Account</h1>
          <UpdateProfileForm />
          <UpdateAccountForm />
          <UpdatePasswordForm />
        </Section>
      </Container>
    );
  }
}
