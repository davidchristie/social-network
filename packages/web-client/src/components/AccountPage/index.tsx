import { Section } from "design-system";
import React from "react";

import Container from "../Container";
import UpdateAccountForm from "../UpdateAccountForm";
import UpdatePasswordForm from "../UpdatePasswordForm";
import UpdateProfileForm from "../UpdateProfileForm";

export default class AccountPage extends React.Component {
  public render () {
    return (
      <Container>
        <Section>
          <h1>Account</h1>
          <UpdateProfileForm />
          <UpdateAccountForm />
          <UpdatePasswordForm />
        </Section>
      </Container>
    );
  }
}
