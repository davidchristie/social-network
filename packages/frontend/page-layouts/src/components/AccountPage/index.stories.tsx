import React from "react";
import { createStories } from "storybook-utilities";
import AccountPage from ".";

createStories("Page Layouts").add(
  "AccountPage",
  () => <AccountPage />
);
