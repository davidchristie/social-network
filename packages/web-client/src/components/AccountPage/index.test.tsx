import React from "react";

import itMatchesSnapshot from "../../testing/itMatchesSnapshot";
import AccountPage from "./index";

describe("AccountPage component", () => {
  itMatchesSnapshot(<AccountPage />);
});
