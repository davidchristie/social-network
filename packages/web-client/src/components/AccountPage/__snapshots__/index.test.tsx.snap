// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountPage component matches snapshot 1`] = `
<div>
  <Section>
    <h1>
      Account
    </h1>
  </Section>
</div>
`;
